web/src/PredictImpactMode.svelte

```diff
@@ -54,12 +55,44 @@
     </p>
     <p>
       Thicker roads have more traffic after edits, relative to the max count for
-      any road: {data.max_count.toLocaleString()}
+      any road: {impactGj.max_count.toLocaleString()}
     </p>
+
+    <SequentialLegendBucketed
+      colorScale={divergingScale}
+      buckets={["0%", "50%", "same", "150%", "200%"]}
+    />
   </div>
 
   <div slot="map">
-    <GeoJSON {data} generateId>
+    <GeoJSON data={neighbourhoods}>
+      <FillLayer
+        {...layerId("neighbourhood-boundaries", false)}
+        filter={["==", ["get", "kind"], "boundary"]}
+        paint={{
+          "fill-color": "grey",
+          "fill-opacity": 0.5,
```

Comment:
I tried a few and settled on this. In a large study area, I just want to remember where the edits have happened / are targeting